### PR TITLE
feat: DAH-2526 Translation for HCBS Units

### PR DESCRIPTION
**Pull request by jimlin-sfgov**
## Description

Add text regarding HCBS Units for The Kelsey listing to the Listing Directory page and Listing Details page.

## Jira ticket

https://sfgovdt.jira.com/browse/DAH-2526

### Version Control

- [x] branch name begins with `angular` if it contains updates to Angular code
- [x] branch name contains the Jira ticket number
- [x] PR name follows `type: TICKET-NUMBER Description` format, e.g. `feat: DAH-123 New Feature`

### Code quality

- [x] [the set of changes is small](https://google.github.io/eng-practices/review/developer/small-cls.html#what-is-small)
- [x] [if the set of changes cannot be small, reviewers have been forewarned](https://google.github.io/eng-practices/review/developer/small-cls.html#cant)
- [x] code meets test coverage thresholds
- [x] code is properly formatted
- [x] code is linted
- [x] code irrelevant to the ticket is not modified e.g. changing indentation due to automated formatting
- [x] automated tests pass
- [x] if the code changes the UI, it matches the UI design exactly

### Review instructions

- [x] instructions specify which environment(s) it applies to
- [x] instructions have already been performed at least once
- [x] instructions can be followed by PA testers
- [x] instructions specify if it can only be followed by an engineer

### Request review

- [x] PR has `needs review` label
- [x] Use `Housing Eng` group to automatically assign reviewers, and/or assign specific engineers
- [x] If time sensitive, notify engineers in Slack

### CODE REVIEW

File: app/assets/json/translations/react/en.json [hunk 1]
```diff
@@ -841,6 +841,7 @@
   "listings.Mobility impairments.title": "Mobility Impairments",
   "listings.Mobility/Hearing/Vision impairments.title": "Mobility, Hearing and/or Vision Impairments",
   "listings.Vision impairments.title": "Vision Impairments",
+  "listings.HCBS Units.title": "In-home Support for a Disability",
```
Comment (alulabeshue-sfgov) [inline app/assets/json/translations/react/en.json:844]: note: this entry is not in alphabetical order

**Review by alulabeshue-sfgov — COMMENTED** [review]
In the ticket's description there is a screen shot that shows this card under the `Priority access to certain units` section while the implementation and review instructions has it under the `Priority Units` section.

File: app/assets/json/translations/react/en.json (hunk 1 above)
Comment (jimlin-sfgov) [reply app/assets/json/translations/react/en.json]: Good catch, they will be auto-sorted when I do the Phrase translation process, but I prefer to keep them ordered beforehand to minimize diffs.

**Comment by jimlin-sfgov** [comment]
> In the ticket's description there is a screen shot that shows this card under the Priority access to certain units section

@alulabeshue-sfgov The screenshot has an incorrectly titled section, and can be ignored. I've made a note of that in the Jira ticket.

**Review by alulabeshue-sfgov — APPROVED** [review]
LG!

**Pull request merged**